src/PluginMediaStreamRenderer.swift

```diff
@@ -2,336 +2,356 @@ import Foundation
 import AVFoundation
 
 class PluginMediaStreamRenderer : NSObject, RTCEAGLVideoViewDelegate {
-	
```

Comment:
Please use same indentation than other swift file for Xcode.
See comment here: https://github.com/cordova-rtc/cordova-plugin-iosrtc/pull/580#issuecomment-710506298